sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -746,6 +780,31 @@ public void processElement(
             tracker.currentRestriction().getFrom(),
             tracker.currentRestriction().getTo());
         Configuration conf = getConfWithModelClass();
+
+        if (useProtoReader) {
+          // Use ProtoParquetReader to read protobuf data.
+          // Derive a Hadoop Path from the file metadata. Adjust as needed.
+          Path path = new Path(file.getMetadata().resourceId().toString());
+
+          // Get the configuration and set the property using the literal.
+          Configuration conf2 = getConfWithModelClass();
+          conf2.set("parquet.proto.ignore.unknown.fields", "TRUE");
```

Comment:
What is the purpose of "ignore.unknown.fields" here? In general we should not set unsafe configure for user if they are not aware of